src/components/action/action.tsx

```diff
@@ -196,9 +224,21 @@ export class Action implements InteractiveComponent, LoadableComponent {
   }
 
   render(): VNode {
-    const { compact, disabled, loading, textEnabled, label, text } = this;
-
-    const ariaLabel = label || text;
+    const {
+      active,
+      compact,
+      disabled,
+      loading,
+      textEnabled,
+      label,
+      text,
+      indicator,
+      indicatorId,
+      buttonId,
+      intlIndicator
+    } = this;
+
+    const ariaLabel = `${label || text}${indicator ? ` (${intlIndicator})` : ""}`;
```

Comment:
nvm. it will be covered [ here](https://github.com/Esri/calcite-components/pull/5471/files#diff-6d2b1195f68eccf57de81ad892c63c9b647f05afca4bb4fd55b42daa9005dcb0R45).

Reply:
Yep. Can you approve?